### PR TITLE
bpf: minor improvements to XDP punt with XFER_PKT_NO_SVC

### PR DESCRIPTION
**Pull request by julianwiedmann**
Just two small improvements to the `XFER_PKT_NO_SVC` processing:
1. move its processing into the `from-netdev` code block that deals with punted packets,
2. add a missing `XFER_PKT_NO_SVC` for non-SVC traffic

### CODE REVIEW

File: bpf/bpf_host.c [hunk 1]
```diff
@@ -1040,16 +1037,23 @@ handle_srv6(struct __ctx_buff *ctx)
 __section("from-netdev")
 int cil_from_netdev(struct __ctx_buff *ctx)
 {
-	__u32 __maybe_unused vlan_id;
```
Comment (sahid) [inline bpf/bpf_host.c:1043]: nit: I would have done this change in a different commit right?

File: bpf/bpf_host.c [hunk 2]
```diff
@@ -1040,16 +1037,23 @@ handle_srv6(struct __ctx_buff *ctx)
...
-
 #ifdef ENABLE_NODEPORT_ACCELERATION
-#ifdef HAVE_ENCAP
 	__u32 flags = ctx_get_xfer(ctx, XFER_FLAGS);
+#ifdef HAVE_ENCAP
```
Comment (sahid) [inline bpf/bpf_host.c:1042]: `if defined(HAVE_ENCAP) && defined(ENABLE_NODEPORT_ACCELERATION)`

**Review by sahid — APPROVED** [review]
Sounds good thank you!

**Review by aspsk — APPROVED** [review]
